### DISCUSSION TITLE
Code examples using Redux, createEntityAdapter and normalizr?

### DISCUSSION THREAD

**Discussion by simplenotezy** [reduxjs/redux#4171] [Help]
Is there any code examples of how to use Redux, createEntityAdapter and normalizr together?

I think the documentation on it is a little lacking of examples and real world code. Nor are there any tutorials or YouTube videos on the subject, so coming with fresh eyes is rather confusing.

**Comment by markerikson** [comment]
Yes, we literally have a section on this in the Usage Guide page in the Redux Toolkit docs :)

https://redux-toolkit.js.org/usage/usage-guide#using-createentityadapter-with-normalization-libraries
